action.yml

```
@@ -83,7 +87,7 @@ runs:
   steps:
     - name: Install action dependencies
       shell: bash
-      run: python3 -m pip install clang-tools cpp-linter
+      run: python3 -m pip install clang-tools cpp-linter==1.4.7
```

I think this isn't really needed because the version pulled from PyPI is always the latest release. But just to avoid having to update this every time we release cpp-linter pkg...
```suggestion
      run: python3 -m pip install clang-tools 'cpp-linter>=1.4.7'
```
In my experience, the quotes are necessary to avoid the `>` from being interpreted as a redirection of stdout in bash.

---

If don't use a specific version here, users can not get a fixed result when cpp-linter package is relased. For example, if a user is using cpp-linter-action@2.2.0, actually, they will have a new feature even they don't upgrade cpp-linter-action, maybe it will break something when cpp-linter package have a problem.

---

If you want to use a fixed version, then you'll have to update this line before every time you publish a release.

We do enough testing to ensure consistent behavior in cpp-linter pkg, and backward compatibility is currently a high priority for me (concerning the cpp-linter pkg).

---

Yes, I have to update here before publishing a release. In my personal, I'd like to use software whose behavior is always fixed, so that I can make sure if they will break my pipeline or introduce any harmful modification.

Recently I used `artifactory-cleanup`, and they released a [problematic package](https://github.com/devopshq/artifactory-cleanup/issues/64) and it was very easy to identify and the example they provided running with docker does not support a fixed version, so I have to create my own docker image to run with a fixed version because people always make mistakes.

---

Ok, this is resolved then.